db/migrate/20211229070656_add_course_to_item.rb

```diff
@@ -0,0 +1,6 @@
+class AddCourseToItem < ActiveRecord::Migration[6.1]
+  def change
+    add_column :items, :course, :string, default: ::Item::MAIN, null: false
```

Comment:
no need to have default value.
and it possible that this will cause irreversible migration error (if Item::MAIN will be deleted or changed)